app/response_planner.py

```diff
@@ -0,0 +1,41 @@
+class LogicalPlanner:

```

Comment:
rename to ResponsePlanner or other

Reply:
I looked into this. Line 225 in c_operation.py requires that the planner have LogicalPlanner as the class name